### ISSUE TITLE
cue/vet: Display obvious errors without -c flag

### ISSUE THREAD

**Issue by asdine** [cue-lang/cue#1928]
**Is your feature request related to a problem? Please describe.**
I'm trying to use cue vet as a linter in a VS Code extension I'm building. Without the -c flag, running `cue vet` doesn't return anything other than `some instances are incomplete; use the -c flag to show errors or suppress this message`, even when there are other errors in the file or package (such as unknown identifiers, not syntax issues).
Enabling the "-c" flag returns lots of "incomplete value" errors for all the imported packages which "hide" the actual user errors.

**Describe the solution you'd like**
I believe the current behavior is useful when we're about to export our configuration but not really useful during development.
Ideally, `cue vet` without a flag should return all "obvious" errors found in the current package ("obvious" then needs to be defined) and not consider non-concrete issues.

**Describe alternatives you've considered**
I considered using -c and ignore all "incomplete value" messages but I'd rather rely on the `cue vet`'s default behavior.

**Additional context**
This behavior is observable when having one or more third party packages in the `cue.mod` directory and importing them in one or more cue files at the root.

**Comment by jpluscplusm** [comment]
This is related to https://github.com/cue-lang/cue/issues/2120. The problem might be helped by using the `-c` flag's negative form (`-c=false`), depending on the user's need.

This negative form isn't well documented just yet ... but we're addressing that at the moment.

**Comment by mvdan** [comment]
I agree with @jpluscplusm. I would suggest that you use `cue vet -c=false` for now, which should do what you need. We're tracking improving the UX for `cue vet` in https://github.com/cue-lang/cue/issues/2120, so please follow that thread instead.
